doc/source/release-notes/Percona-Server-5.6.27-75.0.rst

```diff
@@ -0,0 +1,69 @@
+
+Based on `MySQL 5.6.27 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-27.html>`_, including all the bug fixes in it, |Percona Server| 5.6.27-75.0 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.27-75.0 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.27-75.0>`_.
+
+New Features
+============
+
+ |Percona Server| is now available for *Ubuntu* 15.10 (Wily).
+
+ |TokuDB| MTR tests have been integrated into |Percona Server|.
+ 
+ Linux thread ID is now available in the :table:`PROCESSLIST` table. This feature provides the option to inspect or monitor the |MySQL| server threads with tools external to the server itself, such as `Linux perf <https://perf.wiki.kernel.org/index.php/Main_Page>`_.
+
+ |Percona Server| has now enabled savepoints in triggers and stored functions.
+
+ Variables :variable:`innodb_buffer_pool_populate` and :variable:`numa_interleave` have been removed in favor of upstream implementation of the new `innodb_numa_interleave <http://dev.mysql.com/doc/refman/5.6/en/innodb-parameters.html#sysvar_innodb_numa_interleave>`_ option.
+
+Bugs Fixed
+==========
+
+ Fixed transactional inconsistency with rollback in |TokuDB|  where large inserts within a transaction ended up with some rows committed and some rolled back after executing a rollback. Bug fixed :bug:`1512455`.
+
+ Variable :variable:`tokudb_backup_exclude` was not excluding files correctly. Bug fixed :bug:`1512457`.
+
+ |TokuDB| could crash if run in transaction-isolation was set to ``READ-COMMITTED``. Bug fixed :bug:`1501633`.
+
+ |TokuDB| thread pool names were missing in the ``SHOW ENGINE tokudb STATUS`` which caused duplicate entries. Bug fixed :bug:`1512452`.
+
+ Manipulating the :variable:`innodb_track_redo_log_now` variable dynamically would crash the server if it was started without :variable:`innodb_track_changed_pages` enabled. This variable is available on debug builds only. Bug fixed :bug:`1368530`.
+
+ If the user had duplicate pid-file options in config files when running ``yum upgrade``, the upgrade would stop with error because it would think it found the duplicate pid while it was the same pid specified twice. Bug fixed :bug:`1454917`.
+
+ On some filesystems server would not start if :ref:`changed_page_tracking` feature was enabled and :variable:`innodb_flush_method` variable was set to ``O_DIRECT``. Bugs fixed :bug:`1500720` and :bug:`1498891`.
+
+ When :ref:`user_stats` are enabled, executing any statement of the ``SHOW`` family with non-empty result, would bump :table:`USER_STATISTICS` ``ROWS_FETCHED`` column values erroneously. Bug fixed :bug:`1510953`.
+
+ Fixed server assertion caused by less restrictive locking when :variable:`innodb_fake_changes` was enabled. Bug fixed :bug:`1192898`.
```

Comment:
A write operation with fake changes enabled could cause a server assertion if it followed the pessimistic B-tree update path internally.